shell/edit/namespace.vue

```diff
@@ -169,9 +174,9 @@ export default {
       :value="value"
       :namespaced="false"
       :mode="mode"
+      :extra-columns="['project-col']"
     >
       <template
-        v-if="project"
```

Comment:
If you remove the condition, it will always display the project dropdown. This was meant while editing namespaces created within a project.

You will have then to extend to 2 conditions `v-if="flatView || project"` so it will match both the cases.